.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+<A clear and concise description of what the bug is.>
+
+**To Reproduce**
+<Steps to reproduce the behavior:>
+
+**Expected behavior**
+<A clear and concise description of what you expected to happen.>
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Version Details(please complete the following information):**
+ - Distribution/OS: 
+ - Promscale: 
+ - TimescaleDB:
+
+**Additional context**
```

Comment:
We need a section for `configuration`. In this section, we ask:
1. versions for Promscale, TimescaleDB, Postgres version
2. configuration for Promscale, and Prometheus remote-write/OpenTelemetry/Jaeger as per applicable